bindata/manifests/machine-config/files/configure-switchdev.sh.yaml

```diff
@@ -0,0 +1,56 @@
+mode: 0755
+overwrite: true
+path: "/usr/local/bin/configure-switchdev.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+    input="/etc/switchdev.conf"
+    if [ ! -f $input ]; then
+      echo "File /etc/switchdev.conf not exist."
+      exit
+    fi
+
+    names=()
+
+    while read pci_addr num_vfs
+    do
+      echo "Set $num_vfs VFs on device $pci_addr"
+
+      names+=($(ls /sys/bus/pci/devices/${pci_addr}/net/))
+      # create VFs
+      echo $num_vfs > /sys/bus/pci/devices/${pci_addr}/sriov_numvfs
+    done <"$input"
+
+    # wait for vfs to be ready
+    sleep 5
+    i=0
+    while read pci_addr num_vfs
+    do
+      # unload VF driver
+      VfDirs=$(ls /sys/bus/pci/devices/${pci_addr} | grep virtfn)
+      for VfDir in $VfDirs
+      do
+        VfPciAddr=$(basename "$( readlink -f /sys/bus/pci/devices/${pci_addr}/$VfDir )")
+        echo $VfPciAddr > /sys/bus/pci/drivers/mlx5_core/unbind || true
+
+      done
+
+      # set PF to switchdev mode
+      devlink dev eswitch set pci/${pci_addr} mode switchdev
+
+      # reset the pf name
+      new_name=$(ls /sys/bus/pci/devices/${pci_addr}/net/)
```

Comment:
you should also  enable tc-hw-offload via ethtool

`/usr/sbin/ethtool -K $(new_name) hw-tc-offload on`

Reply:
In my environment, `hw-tc-offload` is on after swichdev enabled. And ethtool is not available in RHCOS.

Reply:
@pliurh 
seems ethtool already exist in 4.7 rhcos 
```
which ethtool 
/usr/sbin/ethtool
```

Reply:
i see,
on older platform it was required to also enable via ethtool.